Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/04-network-hub.md

```diff
@@ -70,6 +70,10 @@ After the domain controller is up and running, you will need to update the DNS o
    az vm restart -g "aks-HUB" -n "server-dev-linux"
```

Comment:
That's not the name of this machine anymore, you updated it to be `client-dev-win` -- Please make sure all deployment guide instructions are tested and updated to reflect your changes.

Reply:
We need not restart the client machine, I will change the MD file and remove this command.
it is not needed anymore.

Reply:
Oh, that's good to know. Nice.

Reply:
But you didn't remove the text that said you're going to restart the jumpbox.

> Restart your domain controller first and then the jumpbox VM to propagate the new DNS settings. Default values for the network resource group and VM names used in example below. 

Please read through the instructions and make sure they are updated to reflect the new changes

Reply:
updated.